packages/aws-lambda/test/ssm_test.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2019
```

Comment:
Fix dates tomorrow

Reply:
You can remove the Instana Copyright, I guess.
It is being removed from the lint rule.
Also, as this is a new file, you should probably change 2021 to 2022 :)